src/envo/envo-edit.owl

```diff
@@ -40861,7 +40861,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501166> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501167> (orthopaedics unit facility)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501167> "A hospital unit facility which is used to provide orthopaedic medicine."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501167> "A hospital unit facility which is used to provide orthopaedic medicine."@en)
```

Comment:
@ddooley Do you want to phase as "provide orthopaedic services"? Or perhaps "in which orthopaedic medicine is performed"?

Reply:
Suggestion accepted!